packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -179,27 +179,30 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     ]
   );
 
-  const messageContentRef = useRef<HTMLDivElement | null>(null);
+  const messageContentAriaText = props.message.content
+    ? props.message.mine
+      ? _formatString(strings.messageContentMineAriaText, {
+          message: props.message.content
+        })
+      : _formatString(strings.messageContentAriaText, {
+          author: `${props.message.senderDisplayName}`,
+          message: props.message.content
+        })
+    : undefined;
 
   const chatMessage = (
     <>
-      <div
-        ref={messageRef}
-        tabIndex={0}
-        onFocus={() => {
-          messageContentRef.current?.focus();
-        }}
-      >
+      <div ref={messageRef} tabIndex={0}>
         <Chat.Message
           data-ui-id="chat-composite-message"
           className={mergeStyles(messageContainerStyle as IStyle)}
           styles={messageContainerStyle}
           content={
-            <div ref={messageContentRef} tabIndex={0}>
+            <div tabIndex={0}>
```

Comment:
Is this working?
During my testing I noticed that if we add a tabIndex here, it gets overriden and set to -1

Reply:
Yes. That is fluentUI n* Chat behaviour. Arrow keys need to be uses to navigate from message to message.

Reply:
I am having a hard time understanding do we need to do anything here and have a chat with fluentUI n* or are things working as expected?

Reply:
Things are working as expected. tabIndex=-1 means that message to message navigation in the n* chat cannot be done by pressing Tab. Arrow key navigation to navigate through messages. This can be verified from the n* Chat documentation sample here https://fluentsite.z22.web.core.windows.net/0.63.0/components/chat/definition.